safetensors/src/tensor.rs

```diff
@@ -871,6 +874,7 @@ impl Display for Dtype {
             Dtype::BF16 => "BF16",
             Dtype::F32 => "F32",
             Dtype::F64 => "F64",
+            Dtype::Complex64 => "Complex64",
```

Comment:
Any way we could keep things in line with what's before `"C64"` just for consistency's sake ?

Hi guys !